README.md

```diff
@@ -1,17 +1,154 @@
+Simmy is a [chaos-engineering](http://principlesofchaos.org/) and fault-injection tool, integrating with the Polly resilience project for .NET.  It is expected to release in early 2019 and will work with Polly v7.0.
 
-## Concept
+# Motivation
 
-_These grabs of slides from our recent conference presentations represent concept - final syntax will be confirmed in documentation but all features are in scope for launch_
+There are a lot of questions when it comes to chaos-engineering and make sure that actually my system is ready to face the worst possibles scenarios.
 
-1 ![simmyslidesnip1](https://user-images.githubusercontent.com/9608723/52367255-8ab7fd80-2a43-11e9-9d55-d9e8abfe14a5.GIF)
+* My system is really resilient enough?
+* Am I really handling the right exceptions/scenarios?
+* Why should I wait for a handled or even and unhandled exception happens in my production environment?
 
-2 ![simmyslidesnip2](https://user-images.githubusercontent.com/9608723/52367266-91df0b80-2a43-11e9-9e01-2e656bf34b31.GIF)
+Using Polly helps me tons to introduce resilience to my project, but I don't want expected or unexpected failures to occur to test it out. My code could be wrongly implemented, testing the scenarios is not straight forward, mocking failure of a dependency for example a cloud SaaS / PaaS service is not always provided straight away.
 
-3 ![simmyslidesnip4](https://user-images.githubusercontent.com/9608723/52367287-a0c5be00-2a43-11e9-8d69-9054ed005881.GIF)
+**How could I simulate chaotics scenarios in my production environment?***
+
+* A way to mock failures of dependencies (any service dependency for example).
+* Takes care of when to fail based on some external factors - maybe global configuration or some rule.
+* Way to revert easily controlling the blast radius
+* Production grade to run this in production system with automation.
+
+# Chaos policies
+
+Simmy offers multiple chaos policies:
+
+|Policy| Premise | Aka| How does the policy works?|
+| ------------- | ------------- |:-------------: |------------- |
+|**[Fault](#fault)**|Some description here.| "Some description here" |  Some description here. |
+|**[Latency](#latency)**|Some description here.| "Some description here" |  Some description here. |
+|**[Behavior](#behavior)**|Some description here.| "Some description here" |  Some description here. |
+
+# Usage
+All chaos policies (Monkey plicies) are designed to inject the behavior randomly (faults, latency or custom behavior), so a Monkey policy allows you to specify an injection rate between 0 and 1 (0-100%) thus, the higher is the injection rate the higher is the probability to inject them. Also it allows you to specify whether or not the random injection is enabled, that way you can release/hold (turn on/off) the monkeys regardless of injection rate you specify, it means, if you specify an injection rate of 100% but you tell to the policy that the random injection is disabled, it will do nothing.
+
```

Comment:
I agree that it would be good to break up this paragraph and present each common parameter with a separate heading.  I did some similar text in the Simmy sample app here: https://github.com/Polly-Contrib/Polly.Contrib.SimmyDemo_WebApi#enabled

(we could re-use the useful parts of that - if useful)